orttraining/orttraining/python/training/ortmodule/_onnx_models.py

```diff
@@ -26,15 +26,13 @@ class ONNXModels:
     1. exported_model: Model that is exported by torch.onnx.export
     2. optimized_model: For eval mode it's exported_model with concrete input shapes set if needed,
                         for training mode, it's optimized model after gradients graph has been built.
-    3. optimized_pre_grad_model: Optimized model before gradient graph is built. It's None for eval mode.
-    4. In addition, ORTModule also saves the execution_model which is the model
+    3. In addition, ORTModule also saves the execution_model which is the model
```

Comment:
Here, `3` is also technically not owned by `ONNXModels`.

Shall we add a comment highlighting that the pre grad model and the execution model will also be saved to the path provided by the user?